airflow-core/src/airflow/cli/commands/config_command.py

```diff
@@ -161,7 +161,6 @@ def message(self) -> str | None:
         old_default="SequentialExecutor",
         new_default="LocalExecutor",
         was_removed=False,
-        breaking=True,
```

Comment:
I don't think changing a default should be breaking.

Reply:
Yep! Not breaking.